benchmark/cmake/modules/AddSwiftBenchmarkSuite.cmake

```diff
@@ -323,6 +332,17 @@ function (swift_benchmark_compile_archopts)
             "-o" "${sibfile}"
             "${source}")
       endif()
+
+      if("${common_options}" MATCHES "save-optimization-record")
+        set(opt_record "${objdir}/${module_name}.opt.yaml")
+        set(opt_viewer "${SWIFT_PATH_TO_LLVM_SOURCE}/tools/opt-viewer/opt-viewer.py")
```

Comment:
Why is this not just exported by LLVM's cmake? Then we can just "suck in" the dependency and not need to worry if upstream changes.